src/map.rs

```diff
@@ -53,6 +53,16 @@ impl<K: Eq + Hash, V> FrozenMap<K, V> {
     }
 }
 
+impl<K, V, S> std::convert::AsMut<HashMap<K, V, S>> for FrozenMap<K, V, S> {
```

Comment:
nit: please put this below the impl below so that it has the right order in the docs